### PR TITLE
Pull request

### PR DESCRIPTION
**Pull request by phi92long**
Everything seems to work fine. Cai ArticleDetailsActivity cua H chua lam no hien het content a?

### CODE REVIEW

File: TopNews/app/src/main/java/com/fisfam/topnews/pojo/Articles.java [hunk 1]
```diff
@@ -107,7 +107,9 @@ public String getUrlToImage() {
     }
 
     public String getPublishedAt() {
-        return publishedAt;
+        //
+        String string = publishedAt.replaceAll("[T|Z]", " ");
+        return string;
```
Comment (hhaietk) [inline TopNews/app/src/main/java/com/fisfam/topnews/pojo/Articles.java:112]: return publishedAt.replaceAll("[T|Z]", " "); ok roi

File: TopNews/app/src/main/res/layout/activity_splash.xml [hunk 2]
```diff
@@ -29,6 +30,6 @@
             android:layout_height="60dp"
             android:src="@drawable/baby_text" />
```
Comment (hhaietk) [inline TopNews/app/src/main/res/layout/activity_splash.xml:31]: thay hinh khac vao di

File: TopNews/app/src/main/AndroidManifest.xml [hunk 3]
```diff
@@ -13,28 +13,25 @@
         android:supportsRtl="true"
         android:theme="@style/AppTheme"
         android:usesCleartextTraffic="true">
-
+        <activity android:name=".CategoryDetailsActivity"></activity>
```
Comment (hhaietk) [inline TopNews/app/src/main/AndroidManifest.xml:16]: add them configchanges voi screenorientation nhu may cai activity kia nua

File: TopNews/app/src/main/java/com/fisfam/topnews/fragment/TopicFragment.java [hunk 4]
```diff
@@ -72,10 +72,10 @@ private void initUiComponents() {
         mRecyclerView = mRootView.findViewById(R.id.category_rv_for_topicfragment);
         mRecyclerView.setHasFixedSize(true);
         mLayoutManager = new LinearLayoutManager(this.getActivity());
-        mAdapter = new CategoryAdapter(CATEGORY_LIST, new CategoryAdapter.OnItemClickListener() {
+        mAdapter = new CategoryAdapter(CATEGORY_LIST, new CategoryAdapter.OnCategoryItemClickListener() {
             @Override
             public void onItemClick(Category category) {
-                Toast.makeText(getContext(),category.getCategoryName(),Toast.LENGTH_SHORT);
+                Toast.makeText(getContext(),category.getCategoryName(),Toast.LENGTH_SHORT).show();
```
Comment (hhaietk) [inline TopNews/app/src/main/java/com/fisfam/topnews/fragment/TopicFragment.java]: chi hien thi Toast thoi a?

File: TopNews/app/src/main/java/com/fisfam/topnews/adapter/HomeAdapter.java [hunk 5]
```diff
@@ -102,7 +105,14 @@ public void onBindViewHolder(@NonNull RecyclerView.ViewHolder holder, int positi
             ItemCategoryViewHolder vh = (ItemCategoryViewHolder) holder;
             vh.recyclerView.setLayoutManager(layoutManager);
             vh.recyclerView.setHasFixedSize(true);
-            CategoryAdapter categoryAdapter = new CategoryAdapter(CATEGORY_LIST);
+            attachSnapHelper(vh.recyclerView);
+            CategoryAdapter categoryAdapter = new CategoryAdapter(CATEGORY_LIST, new CategoryAdapter.OnItemClickListener() {
+                @Override
+                public void onItemClick(Category category) {
+                    Toast.makeText(mContext, "test",Toast.LENGTH_SHORT);
```
Comment (hhaietk) [inline TopNews/app/src/main/java/com/fisfam/topnews/adapter/HomeAdapter.java]: sao cu loan len the nhi, commit nay la luc nao day

**Review by hhaietk — CHANGES_REQUESTED** [review]
read comment

File: TopNews/app/src/main/java/com/fisfam/topnews/fragment/TopicFragment.java (hunk 4 above)
Comment (phi92long) [reply TopNews/app/src/main/java/com/fisfam/topnews/fragment/TopicFragment.java]: cai nay cu r ma. cai moi chuyen den activity moi r

File: TopNews/app/src/main/res/layout/activity_splash.xml (hunk 2 above)
Comment (phi92long) [reply TopNews/app/src/main/res/layout/activity_splash.xml:31]: thay r con j

**Pull request merged**